polkadot/primitives/src/v6/mod.rs

```diff
@@ -14,7 +14,7 @@
 // You should have received a copy of the GNU General Public License
 // along with Polkadot.  If not, see <http://www.gnu.org/licenses/>.
 
-//! `V2` Primitives.
+//! `V6` Primitives.
```

Comment:
Could we remove a re-export of `LOWEST_USER_ID` on the line 47?
Because we are planning to remove it [here](https://github.com/paritytech/polkadot-sdk/pull/1406#issuecomment-1717201652).